apps/openmw/mwgui/tooltips.cpp

```diff
@@ -342,10 +346,35 @@ namespace MWGui
             if (!image)
                 info.icon = "";
             tooltipSize = createToolTip(info);
+            
         }
 
         return tooltipSize;
     }
+    
+    void ToolTips::checkOwned()
+    {
+        if(Settings::Manager::getBool("show owned", "Game"))
+        {
+            MWBase::WindowManager *wm = MWBase::Environment::get().getWindowManager();
+            
+            if(!mFocusObject.isEmpty())
+            {
+                MWWorld::CellRef& cellref = mFocusObject.getCellRef();
+                MWWorld::Ptr ptr = MWBase::Environment::get().getWorld()->getPlayerPtr();
+                MWWorld::Ptr victim;
+                
+                MWMechanics::MechanicsManager* mm = new MWMechanics::MechanicsManager;
```

Comment:
In that case the `isAllowedToUse` function would need to be added to the public interface in MWBase::MechanicsManager.